modules/library/data/src/main/java/org/geotools/data/memory/MemoryEntry.java

```diff
@@ -60,7 +61,7 @@
     MemoryEntry( MemoryDataStore store, SimpleFeatureType schema){
```

Comment:
tricky question, I would love to say "yes", but I understand that some developers have extended this class (for example to preserve feature ids on transaction commit).

It is currently package visible; so they are unlike to get access to it - so subclasses are unlikely to get direct access to it.